src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -70,6 +91,20 @@ CRT_RPC_DECLARE(test_ping_check,
 CRT_RPC_DEFINE(test_ping_check,
 		CRT_ISEQ_TEST_PING_CHECK, CRT_OSEQ_TEST_PING_CHECK)
 
+/* input fields */
+#define CRT_ISEQ_TEST_SWIM_STATUS  				 \
```

Comment:
(style)  please, no space before tabs
(style)  Macros with complex values should be enclosed in parentheses